site/src/components/Primitives/SizeBadge.tsx

```diff
@@ -1,43 +1,39 @@
-import { FiChevronRight } from "solid-icons/fi";
-import { createSignal, onMount, ParentComponent } from "solid-js";
+import { Component, ComponentProps, createSignal, ParentComponent } from "solid-js";
 import SlideModal from "../Modal/SlideModal";
-import BundleSizeModal, { TBundleSizeItem } from "../BundleSizeModal/BundleSizeModal";
-import Stage from "../Stage/Stage";
+import BundleSizeModal from "../BundleSizeModal/BundleSizeModal";
+import { FormattedBytes } from "~/types";
 
-const createFetchSize = (value: string) => {
-  const [size, setSize] = createSignal("");
-  onMount(() => {
-    const fetchValue = async () => {
-      try {
-        const response = await fetch(value);
-        const json = (await response.json()) as { value: string };
-        setSize(json.value);
-      } catch (err) {}
-    };
-    fetchValue();
-  });
-
-  return size;
-};
-
-type SizeProps = {
-  value: string | { gzipped?: string; minified?: string };
-  unit?: string;
-  href: string;
+const getBundleJSHref = ({
+  packageName,
+  exportName,
+  peerDependencies,
+}: {
+  packageName: string;
+  exportName?: string;
+  peerDependencies: string[];
+}) => {
+  const query = encodeURIComponent(`@solid-primitives/${packageName}`);
+  const config = encodeURIComponent(
+    `{"esbuild":${JSON.stringify({ external: peerDependencies })}}`,
+  );
+  return `https://bundlejs.com/?q=${query}${
+    exportName ? `&treeshake=${encodeURIComponent(`[{${exportName}}]`)}` : ""
+  }&config=${config}`;
```

Comment:
This runs unnecessary on client side, since the href has been been set on static build.

Maybe wrap `NoHydration` on `<SizeBadge>` here https://github.com/solidjs-community/solid-primitives/pull/363/files#diff-c2ea4b5863c4324561039c25f2370734e803d621dc447b7e4952151137467f03R124-R129 ?

```js
<Table.TD>
  {pkg.primitives.map(primitive => (
    <SizeBadgeWrapper primitiveName={`${pkg.name}_${primitive.name}`}>
      <NoHydration>
        <SizeBadge
          value={primitive.gzip}
          packageName={pkg.name}
          exportName={primitive.name}
          peerDependencies={pkg.peerDependencies}
        />
      </NoHydration>
    </SizeBadgeWrapper>
  ))}
</Table.TD>
```

Reply:
Theoretically the whole table could be wrapped with no hydration, right?
Maybe not all but right now that content doesn't change.
I'll try experimenting with that.

Reply:
Not the whole table, because `<SizeBadgeWrapper>` has interactivity https://github.com/solidjs-community/solid-primitives/blob/generate-pages/site/src/components/Primitives/SizeBadge.tsx#L47-L56.

Reply:
`<NoHydration>` and `<Hydration>` together?

Reply:
Like nesting `<Hydation>` inside `<NoHydration>`? 
```js
<NoHydration>
  <Table.Table>
    // ...
    <tbody>
      {Object.entries(categories).map(([category, packages]) => (
        <>
         // ...
          {packages.map(pkg => (
              // ...
              <Hydration>
                <Table.TD>
                  <StageBadge value={pkg.stage} />
                </Table.TD>
              </Hydration>
          )}
        </>
      ))}
    </tbody>
  </Table.Table>
</NoHydration>

```

Doesn't work I tried it, even though it does add `data-hk` to `<Table.TD>` which is just a `td` element, but not on `<StageBadge>`'s `button` element.